webpages/js/metaanalyses.js

```diff
@@ -946,7 +946,13 @@
 
     var groupT = _.findEl(plotEl, 'template.group-grapes');
     var groupTooltipsT= _.findEl(plotEl, 'template.group-tooltips');
+
+    var currentGroup = -1;
+
     groups.forEach(function (group, index) {
+
+      currentGroup = currentGroup + 1;
```

Comment:
please use `+= 1` here, it's more readable

also, please consider starting from 0 on line 950 and putting this increment after the value is used on line 998